Xamarin.Forms.Platform.iOS/ViewRenderer.cs

```diff
@@ -40,7 +40,7 @@ public abstract class ViewRenderer<TView, TNativeView> : VisualElementRenderer<T
 		event EventHandler _controlChanging;
 		event EventHandler _controlChanged;
 
-
+		bool _isElementOrControlEmpty => (Element == null || Control == null);
```

Comment:
```suggestion
		bool _isElementOrControlEmpty => Element == null || Control == null;
```

Reply:
Sorry going to remove this to other PR so we can better track it